libcxx/utils/ci/run-buildbot

```diff
@@ -514,7 +534,9 @@ generic-optimized-speed)
 ;;
 generic-pstl-openmp)
     clean
-    generate-cmake -C "${MONOREPO_ROOT}/libcxx/cmake/caches/Generic-pstl-openmp.cmake"
+    generate-cmake-openmp -C "${MONOREPO_ROOT}/libcxx/cmake/caches/Generic-pstl-openmp.cmake"
+    echo "+++ Installing OpenMP and Clang"
+    ${NINJA} -vC "${BUILD_DIR}" install
     check-runtimes
```

Comment:
Do you think this can be done in a more elegant way, @ldionne? By installing Clang and OpenMP as a project rather than a runtime, I got the build-bot to work locally.